cmd/internal/container/mesos/handler.go

```diff
@@ -72,6 +74,20 @@ func newMesosContainerHandler(
 		Paths: cgroupPaths,
 	}
 
+	var resctrlManager *resctrl.IntelRdtManager
```

Comment:
This is `nil` pointer and possible source of SEGFAULT. Consider doing what I did in `stats` package and create no-op implementation of `intelrdt.Manager` interface. It will just do nothing and will save us from lots of bugs.